build.gradle

```diff
@@ -91,7 +91,7 @@ configure(javaProjects) {
             dependency("commons-httpclient:commons-httpclient:3.1")
             dependency("commons-io:commons-io:2.6")
             dependency("commons-validator:commons-validator:1.6")
-            dependencySet(group: "io.grpc", version: "1.12.0") {
```

Comment:
Is this really necessary? Do we run into some kind of trouble by leaving it higher than recommended?

If i remember correctly, we were getting some very weird concurrency bug that would corrupt the network buffers, it was a known issue, and upgrading to 1.12 fixed it.

Reply:
I don't know that it is. You didn't mention that issue yesterday when I asked. It was easier to back down than battle dependency hell internally with the versions but I can remove it and deal with that if there is some functional issue that 1.12 fixed. Since the build passed I didn't think it was an issue. I'll remove this and see if I can resolve the internal versioning.